packages/loading-scripts/src/index.ts

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable no-new */
+/** Loading snippet */
+const sdkBaseUrl = 'https://cdn.rudderlabs.com/beta/v3';
+const sdkName = 'rudder-analytics.min.js';
+const writeKey = '<write-key>';
+const dataPlaneUrl = '<data-plane-url>';
+const options = {
```

Comment:
add another variable called asyncScript with default to true instead of hardcodingbtrue in the scripts. in case a bizzare client does not want async they can set to false